ci/build_wheel.sh

```diff
@@ -48,7 +48,7 @@ fi
 if [[ $PACKAGE_CUDA_SUFFIX == "-cu12" ]]; then
     sed -i "s/cuda-python[<=>\.,0-9a]*/cuda-python>=12.0,<13.0a0/g" ${pyproject_file}
     sed -i "s/cupy-cuda11x/cupy-cuda12x/g" ${pyproject_file}
-    sed -i "/ptxcompiler/d" ${pyproject_file}
+    sed -i "s/ptxcompiler/pynvjitlink-cu12/g" ${pyproject_file}
```

Comment:
I think this might be causing some problems with the pip installation:
```
ERROR: Could not find a version that satisfies the requirement pynvjitlink-cu12-cu12
```
should it be 
```suggestion
    sed -i "s/ptxcompiler/pynvjitlink/g" ${pyproject_file}
```
?

Reply:
Yuck. There's a "ptxcompiler-cu12" being generated in the lines above, and then deleted. This is a hack but it'll be good enough, I guess. I adopted that fix and pushed.